suite-native/alerts/src/alertsAtoms.ts

```diff
@@ -0,0 +1,20 @@
+import { atom } from 'jotai';
+import { RequireAllOrNone } from 'type-fest';
+
+export type AlertType = 'network-error';
```

Comment:
What is this type for?

Reply:
This type was used in the original solution and is deprecated since then. Removed in fixup be3d072d0.